x509/bash.go

```diff
@@ -159,6 +159,13 @@ func SignEl2GoCsr(csrPem string) string {
 	return signCaCsr("el2g-*", csrPem)
 }
 
+func CreateCrl(serials map[string]int) string {
+	if true {
+		panic("This function is not implemented in Bash implementation")
+	}
```

Comment:
Do we even need the `if true` can we not just straight call `panic` ?

Reply:
Without this `if` statement, Golang will inline the panic and produce compiler warnings about unreachable code at random places.
I _believe_ it still inlines this function with the `if true`, but those warnings are then gone.

Reply:
I did not know that, thanks. :+1:

Reply:
> Without this `if` statement, Golang will inline the panic and produce compiler warnings about unreachable code at random places. I _believe_ it still inlines this function with the `if true`, but those warnings are then gone.

I'm not that good in Go, but just a shot: have you tried to use a block code without a clause?
`{ panic(...) }`. IMHO it looks clearer.

Reply:
@MrCry0 let me give it a shot; it looks interesting.

Reply:
It does not help. Golang static analyzer needs a branch.